sdk/Sdk.Generators/DiagnosticDescriptors.cs

```diff
@@ -27,5 +27,46 @@ private static DiagnosticDescriptor Create(string id, string title, string messa
                     messageFormat: "'{0}' class must have a public parameterless constructor.",
                     category: "Startup",
                     severity: DiagnosticSeverity.Error);
+
+        public static DiagnosticDescriptor MultipleBindingsOnAttribute { get; }
```

Comment:
Will add these to the [analyzer docs](https://github.com/Azure/azure-functions-dotnet-worker/tree/main/docs/analyzer-rules) in another PR.